packages/sdks/overrides/angular/src/components/dynamic-div.ts

```diff
@@ -69,21 +69,31 @@ export default class DynamicDiv {
     this.setAttribute(el, 'aria-hidden', this.ariaHidden);
   }
 
-  ngOnChanges() {
+  ngOnChanges(changes) {
     const el = this.v && this.v.nativeElement;
     if (!el) {
       return;
     }
-    this.setAttributes(el, this.attributes);
-    this.setAttributes(el, this.showContentProps);
-    this.setAttribute(el, 'class', this.classProp);
-    this.setAttribute(el, 'style', this.style);
-    this.setAttribute(el, 'builder-parent-id', this.builderParentId);
-    this.setAttribute(el, 'builder-path', this.builderPath);
-    this.setAttribute(el, 'builder-model', this.builderModel);
-    this.setAttribute(el, 'builder-content-id', this.builderContentId);
-    this.setAttribute(el, 'hidden', this.hidden);
-    this.setAttribute(el, 'aria-hidden', this.ariaHidden);
+
+    if (Object.keys(changes).length === 0) {
+      return;
+    }
+
+    if (changes.attributes) this.setAttributes(el, this.attributes);
+    if (changes.showContentProps) this.setAttributes(el, this.showContentProps);
+    if (changes.classProp) this.setAttribute(el, 'class', this.classProp);
+    if (changes.style) this.setAttribute(el, 'style', this.style);
+    if (changes.builderParentId)
+      this.setAttribute(el, 'builder-parent-id', this.builderParentId);
+    if (changes.builderPath)
+      this.setAttribute(el, 'builder-path', this.builderPath);
+    if (changes.builderModel)
+      this.setAttribute(el, 'builder-model', this.builderModel);
+    if (changes.builderContentId)
+      this.setAttribute(el, 'builder-content-id', this.builderContentId);
+    if (changes.hidden) this.setAttribute(el, 'hidden', this.hidden);
+    if (changes.ariaHidden)
```

Comment:
not sure if these extra checks improved things, but I'm keeping them since we override this file anyways